translate/public/locale/en-US/translate.ftl

```diff
@@ -113,22 +113,24 @@ comments-CommentsList--all-comments = ALL COMMENTS
 
 editor-EditorMenu--sign-in-to-translate = <form>Sign in</form> to translate.
 editor-EditorMenu--read-only-localization = This is a read-only localization.
-editor-EditorMenu--button-copy = COPY
+editor-EditorMenu--button-copy = Copy
```

Comment:
We use uppercase intentionally here. We should add a comment explaining why.

See:
https://bugzilla.mozilla.org/show_bug.cgi?id=1666277
https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#css-issues

Reply:
I've reverted my changes on this; sounds like adding that note would be a separate change?